rbac/server/blockchain_transactions/delete_role_admin_transaction.py

```diff
@@ -0,0 +1,48 @@
+# Copyright 2019 Contributors to Hyperledger Sawtooth
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ------------------------------------------------------------------------------
+""" Common Transaction Creation
+"""
+import rethinkdb as r
+from rbac.server.db.db_utils import create_connection
+from rbac.common.role.delete_role_admin import DeleteRoleAdmin
+from rbac.common.sawtooth import batcher
+
+
+async def create_delete_role_admin_txns(key_pair, next_id, txn_list):
```

Comment:
Add arguments (doc strings) to function comment as well please, ty!

Reply:
Do we do that? I thought we just describe the function. I am asking, because I have not seen it in any other tests.